1_fetch.R

```diff
@@ -57,7 +58,21 @@ p1_targets_list <- list(
   tar_target(
     p1_inst_data,
     get_inst_nwis_data(p1_nwis_sites_inst,parameter,start_date=earliest_date,end_date=dummy_date),
-    pattern = map(p1_nwis_sites_inst))
+    pattern = map(p1_nwis_sites_inst)),
   
-)  
+  
+  # Download NLCD datasets 
+  tar_target(
+    NLCD_data, 
```

Comment:
to keep with our target naming convention, add `p1` as a prefix to the target name, and add the file extension as a suffix to the target name. `p1_NLCD_data_<extension>`

Reply:
zip could be the extension.
Comment applies to other targets as well